### PR TITLE
Add Docs for the Term Debt Consolidator

### PR DESCRIPTION
**Pull request by jackbrighton**
## What was done:
- add /docs folder
- add /docs/readme.md with the bulk of the user docs
- add /docs/taxonomy.md just to explain how to add custom taxonomies for those so inclined.
## Why

Docs were needed for this plugin per:
- https://github.com/INN/term-debt-consolidator/issues/3
- https://app.asana.com/0/100231216298059/119540092269697

The new plugin docs are here: https://github.com/INN/term-debt-consolidator/tree/docs/docs

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Term Debt Consolidator can look at your WordPress site's tags to suggest areas for consolidation and deletion and help you identify ways to improve your use of WordPress' taxonomy functionality.
+
+Keep in mind that this process is imperfect and may make nonsensical suggestions from time to time. It's up to you to review the suggestions and approve or deny them.
+
+## Installation
+
+Term Debt Consolidator can be installed like any other WordPress plugin.
+
+1. Install the plugin through the WordPress plugins screen directly, or download the zip file from this Github repo and install manually to your WordPress site.
```
Comment (aschweigert) [inline docs/readme.md]: let's be more specific here, most people will understand what that means, but I'd like the steps to be a bit more explicit

**Comment by jackbrighton** [comment]
The latest commit should clarify things nicely, hopefully.

**Comment by aschweigert** [comment]
still not thrilled with the description of how to install/activate the plugin, also we should standardize this across all of our plugins (because it's the same, more or less)

**Comment by jackbrighton** [comment]
Ok I'll work on that. Any guidance on what it should include?

**Comment by aschweigert** [comment]
I think it should just be a little more explicit, you either upload the files to the folder or you can go to plugins > add new and upload a zip file OR you can search the wp.org directory and add it from there, the instructions here just don't really cover those options sufficiently imo

**Comment by jackbrighton** [comment]
I added to the installation instructions to include getting the plugin from the WP Plugins directory via the dashboard, or alternatively uploading the zip file.

**Pull request merged**